src/app/components/AppLogo/AppLogo.module.css

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: grid;
+  justify-content: center;
+}
+
+.header svg {
+  justify-self: center;
```

Comment:
Not sure if this does anything when you already have the `justify-content: center` in line 3?

Reply:
It actually helps positioning the image as seen in my XD design.